echemdb/data/cv/entry.py

```diff
@@ -207,12 +208,20 @@ def y(self):
         """
         from astropy import units as u
 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A / m2'):
-            return 'j' 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A'):
-            return 'I'
-        else:
-            raise ValueError(f"The variable on the y-axis is not equivalent to 'A / m2' or 'A'.")
+        if not self.data_description.axes:
```

Comment:
I don't think this works. When no axes are specified at all, this will raise an exception.